pyscitt/pyscitt/client.py

```diff
@@ -27,11 +29,14 @@
 class HttpSig(httpx.Auth):
     requires_request_body = True
 
-    def __init__(self, key_id: str, pem_private_key: str):
-        self.key_id = key_id
-        self.private_key = load_pem_private_key(
-            pem_private_key.encode("ascii"), password=None, backend=default_backend()
-        )
+    def __init__(self, key_id: str, pem_private_key: str, akv_configuration:str):
```

Comment:
There should be no AKV-related code in this class. Rather, `key_id` and `pem_private_key` should be replaced by some signer object with a uniform interface for which we have two implementations, local key and AKV.